crypto/crypto.go

```diff
@@ -14,7 +14,7 @@ const (
 )
 
 type SigningKey ed25519.PrivateKey
-type VerifKey ed25519.PublicKey
+type VerifyKey ed25519.PublicKey
```

Comment:
This stuff doesn't look like belongs in this pull.

You should `git rebase -i master` and `d` (drop) that commit.